composeApp/src/commonMain/kotlin/viewModels/RandomizerViewModel.kt

```diff
@@ -68,6 +68,10 @@ object RandomizerViewModel {
         randomizer = generateRandomizer(autoDetect = true)
     }
 
+    fun saveROM(): Any {
```

Comment:
When I said proxy, I meant that you can literally call `RandomizerViewModel.randomizer.saveROM()`, which proxies to the ROM Handler. I don't think we need this function here since `randomizer` is already exposed. Maybe I'll revisit and think about a getter/setter.